tests/Unit/PointwiseFunctions/AnalyticSolutions/GrMhd/Test_AlfvenWave.cpp

```diff
@@ -158,6 +165,21 @@ void test_variables(const DataType& used_for_size) {
           tmpl::list<gr::Tags::SpatialMetric<3, Frame::Inertial, DataType>>{}));
   CHECK_ITERABLE_APPROX(expected_spatial_metric, spatial_metric);
 }
+
+void test_solution() noexcept {
+  grmhd::Solutions::AlfvenWave solution(2.2, 1.23, 0.2, 1.4, {{0.0, 0.0, 2.0}},
```

Comment:
It looks like the B fields here don't have a y-component. I think in general the solution tested should be in full 3D.

Reply:
I agree, but I will leave this to the individual authors of the analytic solutions to update these to test the appropriate cases